server/websocket/receiver.go

```diff
@@ -199,7 +199,7 @@ func (rec *Receiver) fetchOnlyLoop() {
 func (rec *Receiver) fetch() error {
 	fetch := store.FetchRequest{
 		Partition: rec.path.Partition(),
-		MessageC:  make(chan store.MessageAndID, 3),
+		MessageC:  make(chan store.FetchedMessage, 3),
```

Comment:
3 was the initial capacity.Having a bigger capacity there is actually a speed gain since the more messages can be fetched in a concurrent environment.